hazelcast/src/main/java/com/hazelcast/client/impl/management/ManagementCenterService.java

```diff
@@ -311,4 +314,25 @@ public void shutdownMember(Member member) {
                 }
         );
     }
+
+    /**
+     * Gets the latest {@link TimedMemberState} of the member it's called on.
+     */
+    @Nonnull
+    public CompletableFuture<Optional<String>> getTimedMemberState(Member member) {
+        checkNotNull(member);
```

Comment:
I'm not sure whether there's a certain convention, but since this is the entry point for us, I think it makes sense for us to check the parameters for nulls and `@Nonnull` is simply to help the IDE. Those are not used consistently everywhere.